test/utils.sh

```diff
@@ -557,3 +557,77 @@ replace_image_pullspec() {
     exit 2
   fi
 }
+
+# This function will be used by tasks in tekton-integration-catalog
+# Given the output of 'opm render $fbc_fragment' command, this function returns a package name
+# If there is only one 'olm.package', it returns it's name
+# If there are multiple 'olm.package' entries, it returns the one with the highest bundle version in the package’s 'defaultChannel'
```

Comment:
What happens to the rest of the entries, are they being ignored? Are we interested only in highes bundle version one?

Reply:
Yes, we are only interested in the highest bundle version; the other entries are ignored. The purpose of this function is to select a single package and retrieve the highest bundle version from the package's 'defaultChannel'. Since product teams typically work on one bundle version at a time, the highest bundle version should be the one that has not been released yet and will be installed on a cluster as part of the "deploy-fbc" pipeline.